mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/LocalizedStrings.kt

```diff
@@ -17,6 +17,15 @@ interface LocalizedStrings {
   val authenticateSwitchToRegister: String
   val authenticateErrorFailure: String
 
+  /** Profile Strings */
```

Comment:
This is some documentation, not a comment 😄. Also, we probably don't need to document all these strings just yet.

```suggestion
```